File: 01-Assignment-02-02-2026/Using_C/program_2.c
/* Program 2: Convert temperature from Celsius to Fahrenheit */

#include <stdio.h>

int main() {
    float c,f;
    printf("Enter Temperture in Celsius : ");
    scanf("%f",&c);
    f = (c* 9/5) + 32 ;
    printf("%.2f Degree Celsius => %.2f Fahrenheit\n",c,f);
    return 0;
}

File: 01-Assignment-02-02-2026/Using_Java/Program2.java
/*
Program 2
Java program to convert temperature from Celsius to Fahrenheit
*/

import java.util.Scanner;

public class Program2 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter Temperature in Degree Celsius  : ");
        double C = scanner.nextDouble();

        double F = 1.8 * C + 32;
        System.out.println(C + " Degree Celsius is " + F + " Fahrenheit");

        scanner.close();
    }

}
